cosmos/dbt/graph.py

```diff
@@ -261,12 +261,13 @@ def run_command(
 ) -> str:
     """Run a command either with dbtRunner or Python subprocess, returning the stdout."""
 
-    runner = "dbt Runner" if dbt_runner.is_available() else "Python subprocess"
+    use_dbt_runner = invocation_mode == InvocationMode.DBT_RUNNER and dbt_runner.is_available()
```

Comment:
looks like the variable assignment to `invocation_mode` maybe now redundant here? don't see it getting used in the method

Reply:
@pankajkoti so sorry, I missed your comment before merging. `invocation_mode` is an argument in the scope of this function. How do you think we could improve this?

Reply:
sorry I had misread the expression. We are not actually assigning to invocation_mode but using it for evaluation. Apologies, please ignore 🙇🏽